content/guides/autodiscovery.md

```diff
@@ -48,155 +86,169 @@ By default, the Datadog Agent includes Autodiscovery support for:
+Autodiscovery supports Consul, etcd, and Zookeeper as template sources. To use a KV store, configure its parameters in `datadog.conf` or in environment variables passed to docker-dd-agent when starting the container.
 
-The default management port for the rabbitmq image is `15672` with index 4 in the list (starting from 0), so the template variable needs to be `%%port_4%%`.
+#### Configure in `datadog.conf`
 
-As of version `5.8.3` of the Datadog Agent, you can also use keys as a suffix when a variable contains a dictionary. This is particularly useful to select an IP address for a container that has several networks attached.
+In the `datadog.conf` file, set the `sd_config_backend`, `sd_backend_host`, and `sd_backend_port` options to, respectively, the backend type—`etcd`, `consul`, or `zookeeper`—and the IP address and port where your KV store is listening:
 
-As an example if the rabbitmq container mentioned above is available over two networks `bridge` and `swarm`, using `%%host_swarm%%` will pick the IP address from the swarm network.
-Note that for the `host` variable if several networks are found and no key is passed the agent attempts to use the default `bridge` network.
+~~~
+# For now only Docker is supported so you just need to un-comment this line.
+service_discovery_backend: docker
 
-## Configuration templates with key-value stores
+# Define which key/value store must be used to look for configuration templates.
+# Default is etcd. Consul is also supported.
+sd_config_backend: etcd
 
-Using Autodiscovery with the configuration templates in the Datadog Agent  `conf.d/auto_conf` directory is a straightforward process, though managing your templates and copying them into the Datadog Agent container (or building your own Datadog Agent container to include custom configuration templates) can make scaling this process difficult.
+# Settings for connecting to the backend. These are the default, edit them if you run a different config.
+sd_backend_host: 127.0.0.1
+sd_backend_port: 4001
 
-To make configuration template management easier, you can use etcd or Consul, two popular distributed key-value stores, as a repository for your templates.
+# By default, the agent will look for the configuration templates under the
+# `/datadog/check_configs` key in the back-end.
+# If you wish otherwise, uncomment this option and modify its value.
+# sd_template_dir: /datadog/check_configs
 
-First you'll need to configure etcd or Consul as your Autoiscovery backend by either updating the `datadog.conf` file or passing the settings as environment variables when starting the Datadog Agent service.
+# If you Consul store requires token authentication for service discovery, you can define that token here.
+# consul_token: f45cbd0b-5022-samp-le00-4eaa7c1f40f1
+~~~
 
-### Configuring etcd or Consul in `datadog.conf`
+If you are using Consul and the Consul cluster requires token authentication, set `consul_token`.
```

Comment:
FWI, 5.15 will support user/pass auth for etcd. See https://github.com/DataDog/dd-agent/pull/3357